synapse_data_warehouse/synapse/dynamic_tables/V2.22.0__accessrequirement_latest.sql

```diff
@@ -0,0 +1,26 @@
+use schema {{database_name}}.synapse; --noqa: JJ01,PRS,TMP,CP01
+
+CREATE DYNAMIC TABLE IF NOT EXISTS ACCESSREQUIREMENT_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS
+    WITH latest_unique_rows AS (
+        SELECT
+            accessrequirementsnapshots.*,
+        FROM
+            {{database_name}}.synapse_raw.accessrequirementsnapshots --noqa: TMP
+        WHERE
```

Comment:
@jaymedina , I'll defer to @philerooski first to see if there are any issues with this query.  Hint: it's all about _when_ the DELETE event is removed.

Note, there may not be any issues, i just haven't dove into it.